handler.js

```diff
@@ -2,7 +2,7 @@
 function runTest(event, context, cb){ 
 
 	const exec = require('child_process').exec;
-	exec("java -jar fitnesse-standalone.jar -d 'FitnesseRoot' -c '" + event.test + "?test&format=xml&includehtml' -b '" + event.test + ".xml' -p 8080", (error, stdout, stderr) => {
+	exec("cp -a FitNesseRoot /tmp; chmod 755 /tmp; java -jar fitnesse-standalone.jar -d '/tmp' -c '" + event.test + "?test&format=xml&includehtml' -b '/tmp/" + event.test + ".xml' -p 8002; cat '/tmp/" + event.test + ".xml'", (error, stdout, stderr) => {
```

Comment:
need to check how it works on sequential runs (when aws container reused from previous lambda run)